.gitignore

```diff
@@ -7,7 +7,7 @@
 
 #ignore any files in config/, except those in subdirectories.
 /config/*
-!/config/*/*
+!config/**/*/
```

Comment:
But that's already what this line does. Look at the history of the files in the example folder, and you'll see they've been changed plenty of times. This change changes nothing...